ex02.ex

```diff
@@ -53,12 +60,12 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
-
+  first2equal = fn 
+                  [a | [a | b]] -> true
```

Comment:
This is good, but it is more idiomatic just to write

~~~ elixir
[ a, a | _tail ] -> true
~~~

Also, your code defines b, and never uses it, resulting in a warning.

The next line generates warnings for a, b, and c.

I'm deducting 1 for this, as I'd like to see no warnings at all.

### -1